staging/src/k8s.io/code-generator/generate-internal-groups.sh

```diff
@@ -47,15 +47,15 @@ EXT_APIS_PKG="$4"
 GROUPS_WITH_VERSIONS="$5"
 shift 5
 
-go install ${GOFLAGS:-} ./$(dirname "${0}")/cmd/{defaulter-gen,conversion-gen,client-gen,lister-gen,informer-gen,deepcopy-gen}
```

Comment:
why this?

Reply:
ref: https://github.com/kubernetes/kubernetes/pull/73246#discussion_r250833676 , so we think it maybe the best answer to pass shellcheck here.

Reply:
makes sense